unkeyed/unkey
# fix: resolve pricing issues

## What does this PR do?



Fixes #1508 

*If there is not an issue for this, please create one first. This is used to tracking purposes and also helps use understand why this PR exists*



## Type of change




## How should this be tested?



- Test A: Free plan max quota
In a new free workspace, set the current api keys to 101, the hard cap banner should no longer appear.
- Test B: Pro subscription invoicing
In a pro workspace, set the current verifications to 150k and generate an invoice. The user should not get charged an additional amount.

## Checklist



### Required


### Appreciated
